code/modules/client/preference_setup/loadout/loadout_uniform.dm

```diff
@@ -5,10 +5,35 @@
 	slot = slot_w_uniform
 	sort_category = "Uniforms and Casual Dress"
 
+/datum/gear/uniform/bathrobe
+	display_name = "bathrobe"
```

Comment:
I was once told by lore writers (I don't recall who it was, I'm afraid) that the NSS Aurora can and may house people who can live on there, if at the very least for a temporary period like a dormitory.
Similarly, one of the reasons for adding in uniforms was to allow people to bring these in their backpacks rather than start off wearing them—and that's just assuming the narrative that they are going to the station, yet alone already being on it. I should also point out that both the station and the loadout from which characters in-universe may bring include other forms of toiletries such as towels and toothpaste.